core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.html

```diff
@@ -714,5 +711,10 @@ <h2>{{ noDataMessage }}</h2>
       display: flex;
       justify-content: space-around;
     }
+
+    .attribute-key {
+      min-width: 50%;
   }
+
+}
```

Comment:
Lines 717 and 719 still need 2 spaces of indentation.

Reply:
added